apps/api/src/controllers/collaboratorsController.ts

```diff
@@ -99,6 +99,37 @@ export const createCollaborators = async ({
 	return result;
 };
 
+/**
+ * Delete a selected collaborator by ID
+ * @param application_id - ID of related application record to associate with Collaborators
+ * @param user_id - ID of Applicant updating the application
+ * @param collaborator_id - ID of Collaborator to delete
+ * @returns Success with Collaborator data record / Failure with Error.
+ */
+export const deleteCollaborator = async ({ application_id, id }: { application_id: number; id: number }) => {
+	const database = getDbInstance();
+	const collaboratorsRepo: CollaboratorsService = collaboratorsSvc(database);
+	const applicationRepo: ApplicationService = applicationSvc(database);
+
+	const applicationResult = await applicationRepo.getApplicationById({ id: application_id });
+
+	if (!applicationResult.success) {
+		return applicationResult;
+	}
+
+	const application = applicationResult.data;
+
+	if (!(application.state === 'DRAFT')) {
+		return failure(`Can only add Collaborators when Application is in state DRAFT`, 'InvalidState');
```

Comment:
Probably don't want to include the valid state list in our error message, won't remember to update this message if the state model ever changes.

Reply:
Just saw this, I'll revisit in next PR